### ISSUE TITLE
crash when opening devtools on macOS 10.12

### ISSUE THREAD

**Issue by ghostoy** [nwjs/nw.js#5497]
Reported by @proclaim in #4437 

> Using Sierra, I'm using the latest v0.18.7, with node 7.1.0, still crash when I try to open dev tool

> here's the download, FLUX studio 0.5.0
> https://flux3dp.com/downloads
>
> however, when you hover to the top left icon and use "inspect" to call out the devtool seems fine
ps. use ctrl + alt + d to call out the devtool

This is the crash dump

```
Operating system: Mac OS X
                  10.12.2 16C48b
CPU: amd64
     family 6 model 58 stepping 9
     8 CPUs

Crash reason:  EXC_BAD_ACCESS / KERN_INVALID_ADDRESS
Crash address: 0x10

Thread 0 (crashed)
 0  AppKit + 0x5da311
    rax = 0x0000000000000028   rdx = 0x0000000000000000
    rcx = 0x0000000000000010   rbx = 0x00007fffc5414b40
    rsi = 0x00007fffaee56b13   rdi = 0x0000000000000000
    rbp = 0x00007fff521d5100   rsp = 0x00007fff521d50d0
     r8 = 0x0000000000000060    r9 = 0x0000000000000000
    r10 = 0x00007fffcafd6248   r11 = 0x00007fb588047b30
    r12 = 0x00006100001276c0   r13 = 0x00006100001276c0
    r14 = 0x0000000000040000   r15 = 0x0000610000127580
    rip = 0x00007fffae7d2311
    Found by: given as instruction pointer in context
 1  AppKit + 0x7bfef0
    rbp = 0x00007fff521d52a0   rsp = 0x00007fff521d5110
    rip = 0x00007fffae9b7ef0
    Found by: previous frame's frame pointer
 2  nwjs Framework!__34-[BrowserCrApplication sendEvent:]_block_invoke [chrome_browser_application_mac.mm : 343 + 0x23]
    rbp = 0x00007fff521d52f0   rsp = 0x00007fff521d52b0
    rip = 0x0000000117b868bc
    Found by: previous frame's frame pointer
 3  nwjs Framework!base::mac::CallWithEHFrame(void () block_pointer) + 0xa
    rbx = 0x00006100001276c0   rbp = 0x00007fff521d5300
    rsp = 0x00007fff521d5300   r12 = 0x0000000000000008
    r14 = 0x000061800003b220   r15 = 0x0000000000000009
    rip = 0x00000001140e8d1a
    Found by: call frame info
 4  nwjs Framework!-[BrowserCrApplication sendEvent:] [chrome_browser_application_mac.mm : 327 + 0x5]
    rbx = 0x00006100001276c0   rbp = 0x00007fff521d5340
    rsp = 0x00007fff521d5310   r12 = 0x0000000000000008
    r14 = 0x000061800003b220   r15 = 0x0000000000000009
    rip = 0x0000000117b86808
    Found by: call frame info
 5  nwjs Framework!-[CommandDispatcher redispatchKeyEvent:] [command_dispatcher.mm : 111 + 0x10]
    rbx = 0x00006100001276c0   rbp = 0x00007fff521d53c0
    rsp = 0x00007fff521d5350   r12 = 0x0000000000000008
    r14 = 0x000061800003b220   r15 = 0x0000000000000009
    rip = 0x00000001148baae6
    Found by: call frame info
 6  nwjs Framework!+[BrowserWindowUtils handleKeyboardEvent:inWindow:] [browser_window_utils.mm : 77 + 0x13]
    rbx = 0x00007fffc5414b01   rbp = 0x00007fff521d53f0
    rsp = 0x00007fff521d53d0   r12 = 0x00006080001291a0
    r13 = 0x00007fffc5414b40   r14 = 0x00007fb587e34b70
    r15 = 0x00006080001291a0   rip = 0x0000000118a016bf
    Found by: call frame info
 7  nwjs Framework!BrowserWindowCocoa::HandleKeyboardEvent(content::NativeWebKeyboardEvent const&) [browser_window_cocoa.mm : 734 + 0x13]
    rbx = 0x00007fff521d5448   rbp = 0x00007fff521d5430
    rsp = 0x00007fff521d5400   r12 = 0x00006080001291a0
    r13 = 0x00007fffc5414b40   r14 = 0x0000618000283de0
    r15 = 0x000000011acb6ff8   rip = 0x00000001189f4542
    Found by: call frame info
 8  nwjs Framework!content::InputRouterImpl::ProcessKeyboardAck(blink::WebInputEvent::Type, content::InputEventAckState, ui::LatencyInfo const&) [input_router_impl.cc : 585 + 0x13]
    rbx = 0x00007fff521d5448   rbp = 0x00007fff521d56b0
    rsp = 0x00007fff521d5440   r12 = 0x00007fb5889b5c00
    r13 = 0x00007fb5889b5c00   r14 = 0x0000000000000002
    r15 = 0x00007fff521d54a8   rip = 0x00000001153a074c
    Found by: call frame info
 9  nwjs Framework!content::InputRouterImpl::ProcessInputEventAck(blink::WebInputEvent::Type, content::InputEventAckState, ui::LatencyInfo const&, unsigned int, content::InputRouterImpl::AckSource) [input_router_impl.cc : 547 + 0xd]
    rbx = 0x0000000000000007   rbp = 0x00007fff521d5770
    rsp = 0x00007fff521d56c0   r12 = 0x00007fb589140a00
    r13 = 0x0000000000000000   r14 = 0x0000000000000002
    r15 = 0x000000011ad80b54   rip = 0x00000001153a0347
    Found by: call frame info
10  nwjs Framework!content::InputRouterImpl::OnInputEventAck(content::InputEventAck const&) [input_router_impl.cc : 471 + 0xe]
    rbx = 0x00007fff521d57b8   rbp = 0x00007fff521d57a0
    rsp = 0x00007fff521d5780   r12 = 0x00006000002d41b8
    r13 = 0x00007fb589140a00   r14 = 0x00007fb589140a00
    r15 = 0x0000000000000000   rip = 0x000000011539f124
    Found by: call frame info
11  nwjs Framework!bool IPC::MessageT<InputHostMsg_HandleInputEvent_ACK_Meta, std::__1::tuple<content::InputEventAck>, void>::Dispatch<content::InputRouterImpl, content::InputRouterImpl, void, void (content::InputRouterImpl::*)(content::InputEventAck const&)>(IPC::Message const*, content::InputRouterImpl*, content::InputRouterImpl*, void*, void (content::InputRouterImpl::*)(content::InputEventAck const&)) [tuple.h : 140 + 0x1f]
    rbx = 0x00007fff521d57b0   rbp = 0x00007fff521d59f0
    rsp = 0x00007fff521d57b0   r12 = 0x00006000002d41b8
    r13 = 0x00007fb589140a00   r14 = 0x000000011539f080
    r15 = 0x0000000000000000   rip = 0x000000011539ef8e
    Found by: call frame info
12  nwjs Framework!content::InputRouterImpl::OnMessageReceived(IPC::Message const&) [input_router_impl.cc : 257 + 0x1a]
    rbx = 0x00006000002d41b8   rbp = 0x00007fff521d5b40
    rsp = 0x00007fff521d5a00   r12 = 0x00007fb587d0dd50
    r13 = 0x00007fff521d6110   r14 = 0x00007fb589140a00
    r15 = 0x00007fff521d5a00   rip = 0x000000011539ec0f
    Found by: call frame info
13  nwjs Framework!content::RenderWidgetHostImpl::OnMessageReceived(IPC::Message const&) [render_widget_host_impl.cc : 482 + 0x8]
    rbx = 0x00007fb587f48fa0   rbp = 0x00007fff521d5df0
    rsp = 0x00007fff521d5b50   r12 = 0x00007fb587d0dd50
    r13 = 0x00007fff521d6110   r14 = 0x00006000002d41b8
    r15 = 0x000000011ad80b36   rip = 0x00000001153f71c0
    Found by: call frame info
14  nwjs Framework!content::RenderProcessHostImpl::OnMessageReceived(IPC::Message const&) [render_process_host_impl.cc : 1786 + 0x8]
    rbx = 0x00007fb587f039a0   rbp = 0x00007fff521d5f50
    rsp = 0x00007fff521d5e00   r12 = 0x00007fb587d0dd50
    r13 = 0x00007fff521d6110   r14 = 0x00006000002d41b8
    r15 = 0x000000011ad80b36   rip = 0x00000001153e5379
    Found by: call frame info
15  nwjs Framework!IPC::ChannelProxy::Context::OnDispatchMessage(IPC::Message const&) [ipc_channel_proxy.cc : 284 + 0x8]
    rbx = 0x00007fb587f0a590   rbp = 0x00007fff521d5f70
    rsp = 0x00007fff521d5f60   r12 = 0x00007fb587d0dd50
    r13 = 0x00007fff521d6110   r14 = 0x00006000002d41b8
    r15 = 0x000000011ad80b36   rip = 0x0000000114ec6628
    Found by: call frame info
16  nwjs Framework!base::debug::TaskAnnotator::RunTask(char const*, base::PendingTask const&) [callback.h : 389 + 0x4]
    rbx = 0x00007fff521d5f88   rbp = 0x00007fff521d6030
    rsp = 0x00007fff521d5f80   r12 = 0x00007fb587d0dd50
    r13 = 0x00007fff521d6110   r14 = 0x0000000118f3121d
    r15 = 0x000000011ad80b36   rip = 0x00000001140d080b
    Found by: call frame info
17  nwjs Framework!base::MessageLoop::RunTask(base::PendingTask const&) [message_loop.cc : 493 + 0xf]
    rbx = 0x000000011add840c   rbp = 0x00007fff521d60e0
    rsp = 0x00007fff521d6040   r12 = 0x00007fb587d0dd28
    r13 = 0x0000000118fb6b5e   r14 = 0x00007fff521d6110
    r15 = 0x00007fb587d0dc10   rip = 0x00000001140f3cdc
    Found by: call frame info
18  nwjs Framework!base::MessageLoop::DeferOrRunPendingTask(base::PendingTask const&) [message_loop.cc : 502 + 0xb]
    rbx = 0x00007fb587d0dc10   rbp = 0x00007fff521d6100
    rsp = 0x00007fff521d60f0   r12 = 0xa0a0a0a0a0a0a0a1
    r13 = 0x00007fff521d6110   r14 = 0x00007fff521d6110
    r15 = 0x00007fb587d0dc58   rip = 0x00000001140f3fec
    Found by: call frame info
19  nwjs Framework!base::MessageLoop::DoWork() [message_loop.cc : 624 + 0xb]
    rbx = 0x00007fb587d0dc10   rbp = 0x00007fff521d6190
    rsp = 0x00007fff521d6110   r12 = 0xa0a0a0a0a0a0a0a1
    r13 = 0x00007fff521d6110   r14 = 0x00007fff521d6160
    r15 = 0x00007fb587d0dc58   rip = 0x00000001140f434b
    Found by: call frame info
20  nwjs Framework!base::MessagePumpCFRunLoopBase::RunWork() [message_pump_mac.mm : 330 + 0x9]
    rbx = 0x00006000002c1ea0   rbp = 0x00007fff521d61c0
    rsp = 0x00007fff521d61a0   r12 = 0x000060000017e480
    r13 = 0x00007fffcb5b0622   r14 = 0x0000608000044f80
    r15 = 0x000060800017d988   rip = 0x00000001140f674d
    Found by: call frame info
21  nwjs Framework!base::mac::CallWithEHFrame(void () block_pointer) + 0xa
    rbx = 0x0000608000188d70   rbp = 0x00007fff521d61d0
    rsp = 0x00007fff521d61d0   r12 = 0x000060000017e480
    r13 = 0x00007fffcb5b0622   r14 = 0x000060000017e498
    r15 = 0x000060800017d988   rip = 0x00000001140e8d1a
    Found by: call frame info
22  nwjs Framework!base::MessagePumpCFRunLoopBase::RunWorkSource(void*) [message_pump_mac.mm : 306 + 0x5]
    rbx = 0x0000608000188d70   rbp = 0x00007fff521d6210
    rsp = 0x00007fff521d61e0   r12 = 0x000060000017e480
    r13 = 0x00007fffcb5b0622   r14 = 0x000060000017e498
    r15 = 0x000060800017d988   rip = 0x00000001140f6154
    Found by: call frame info
23  CoreFoundation + 0xa48d1
    rbx = 0x0000608000188d70   rbp = 0x00007fff521d6220
    rsp = 0x00007fff521d6220   r12 = 0x000060000017e480
    r13 = 0x00007fffcb5b0622   r14 = 0x000060000017e498
    r15 = 0x000060800017d988   rip = 0x00007fffb072d8d1
    Found by: call frame info
24  CoreFoundation + 0x85c6c
    rbp = 0x00007fff521d6280   rsp = 0x00007fff521d6230
    rip = 0x00007fffb070ec6c
    Found by: previous frame's frame pointer
25  CoreFoundation + 0x85156
    rbp = 0x00007fff521d6f70   rsp = 0x00007fff521d6290
    rip = 0x00007fffb070e156
    Found by: previous frame's frame pointer
26  CoreFoundation + 0x84b54
    rbp = 0x00007fff521d7000   rsp = 0x00007fff521d6f80
    rip = 0x00007fffb070db54
    Found by: previous frame's frame pointer
27  HIToolbox + 0x30b2c
    rbp = 0x00007fff521d7040   rsp = 0x00007fff521d7010
    rip = 0x00007fffafc98b2c
    Found by: previous frame's frame pointer
28  HIToolbox + 0x30961
    rbp = 0x00007fff521d70c0   rsp = 0x00007fff521d7050
    rip = 0x00007fffafc98961
    Found by: previous frame's frame pointer
29  HIToolbox + 0x30796
    rbp = 0x00007fff521d70e0   rsp = 0x00007fff521d70d0
    rip = 0x00007fffafc98796
    Found by: previous frame's frame pointer
30  AppKit + 0x46bfc
    rbp = 0x00007fff521d7500   rsp = 0x00007fff521d70f0
    rip = 0x00007fffae23ebfc
    Found by: previous frame's frame pointer
31  AppKit + 0x7c12e3
    rbp = 0x00007fff521d77b0   rsp = 0x00007fff521d7510
    rip = 0x00007fffae9b92e3
    Found by: previous frame's frame pointer
32  AppKit + 0x3b57d
    rbp = 0x00007fff521d78a0   rsp = 0x00007fff521d77c0
    rip = 0x00007fffae23357d
    Found by: previous frame's frame pointer
33  nwjs Framework!base::MessagePumpNSApplication::DoRun(base::MessagePump::Delegate*) [message_pump_mac.mm : 669 + 0xd]
    rbp = 0x00007fff521d7900   rsp = 0x00007fff521d78b0
    rip = 0x00000001140f6ede
    Found by: previous frame's frame pointer
34  nwjs Framework!base::MessagePumpCFRunLoopBase::Run(base::MessagePump::Delegate*) [message_pump_mac.mm : 238 + 0xc]
    rbx = 0x00006000002c1ea0   rbp = 0x00007fff521d7930
    rsp = 0x00007fff521d7910   r12 = 0x0000000000000000
    r13 = 0x00006000000fdd00   r14 = 0x00007fb587d0dc10
    r15 = 0x0000000000000000   rip = 0x00000001140f65a4
    Found by: call frame info
35  nwjs Framework!base::RunLoop::Run() [run_loop.cc : 35 + 0x5]
    rbx = 0x00007fff521d7980   rbp = 0x00007fff521d7970
    rsp = 0x00007fff521d7940   r12 = 0x0000000000000000
    r13 = 0x00006000000fdd00   r14 = 0x00007fff521d7940
    r15 = 0x00007fb587c02ba0   rip = 0x0000000114111c41
    Found by: call frame info
36  nwjs Framework!ChromeBrowserMainParts::MainMessageLoopRun(int*) [chrome_browser_main.cc : 1958 + 0x8]
    rbx = 0x00007fff521d7980   rbp = 0x00007fff521d79d0
    rsp = 0x00007fff521d7980   r12 = 0x0000000000000000
    r13 = 0x00006000000fdd00   r14 = 0x00007fb587c02a18
    r15 = 0x00007fb587c02ba0   rip = 0x0000000117b8adb5
    Found by: call frame info
37  nwjs Framework!content::BrowserMainLoop::RunMainMessageLoopParts() [browser_main_loop.cc : 942 + 0x3]
    rbx = 0x000000011ad80b30   rbp = 0x00007fff521d7a00
    rsp = 0x00007fff521d79e0   r12 = 0x0000000000000000
    r13 = 0x00006000000fdd00   r14 = 0x00007fb587c02a00
    r15 = 0x00007fb589004600   rip = 0x00000001151beb84
    Found by: call frame info
38  nwjs Framework!content::BrowserMainRunnerImpl::Run() [browser_main_runner.cc : 155 + 0x5]
    rbx = 0x0000618000027380   rbp = 0x00007fff521d7a20
    rsp = 0x00007fff521d7a10   r12 = 0x0000000000000000
    r13 = 0x00006000000fdd00   r14 = 0x00000000ffffffff
    r15 = 0x00007fb589004600   rip = 0x00000001151c1042
    Found by: call frame info
39  nwjs Framework!content::BrowserMain(content::MainFunctionParams const&) [browser_main.cc : 46 + 0x9]
    rbx = 0x0000618000027380   rbp = 0x00007fff521d7a70
    rsp = 0x00007fff521d7a30   r12 = 0x0000000000000000
    r13 = 0x00006000000fdd00   r14 = 0x00000000ffffffff
    r15 = 0x00007fb589004600   rip = 0x00000001151bb10d
    Found by: call frame info
40  nwjs Framework!content::ContentMainRunnerImpl::Run() [content_main_runner.cc : 788 + 0x5]
    rbx = 0x00007fff521d7ab0   rbp = 0x00007fff521d7b90
    rsp = 0x00007fff521d7a80   r12 = 0x0000000000000000
    r13 = 0x00006000000fdd00   r14 = 0x00000001193a40ab
    r15 = 0x00006080000a90c0   rip = 0x0000000117b50910
    Found by: call frame info
41  nwjs Framework!content::ContentMain(content::ContentMainParams const&) [content_main.cc : 22 + 0x9]
    rbx = 0x00006080000a90c0   rbp = 0x00007fff521d7bb0
    rsp = 0x00007fff521d7ba0   r12 = 0x0000000000000001
    r13 = 0x00007fb588800a00   r14 = 0x00000000ffffffff
    r15 = 0x00007fff521d7bc0   rip = 0x0000000117b4fb06
    Found by: call frame info
42  nwjs Framework!ChromeMain [chrome_main.cc : 84 + 0x5]
    rbx = 0x0000000000000001   rbp = 0x00007fff521d7c30
    rsp = 0x00007fff521d7bc0   r12 = 0x0000000000000001
    r13 = 0x00007fb588800a00   r14 = 0x00007fff521d7ca0
    r15 = 0x00007fff521d7bc0   rip = 0x00000001140567ea
    Found by: call frame info
43  nwjs!main [chrome_exe_main_mac.c : 87 + 0x8]
    rbx = 0x00007fb587e00001   rbp = 0x00007fff521d7c80
    rsp = 0x00007fff521d7c40   r12 = 0x0000000000000001
    r13 = 0x00007fb588800a00   r14 = 0x00007fff521d7ca0
    r15 = 0x00006000002c0cb0   rip = 0x000000010da28d52
    Found by: call frame info
44  nwjs!start + 0x34
    rbx = 0x0000000000000000   rbp = 0x00007fff521d7c90
    rsp = 0x00007fff521d7c90   r12 = 0x0000000000000000
    r13 = 0x0000000000000000   r14 = 0x0000000000000000
    r15 = 0x0000000000000000   rip = 0x000000010da28b34
    Found by: call frame info
```

**Comment by ghostoy** [comment]
The simplest way to reproduce the issue.

```js
window.onkeypress = (e)=>nw.Window.open('index.html');
```

Pressing any **two** modifiers + key will crash.

**Comment by ghostoy** [comment]
@proclaim Simple workaround for this issue: `e.preventDefault()`. Try this workaround before we can fix it.

**Comment by ghostoy** [comment]
@rogerwang Could you please take a look at this crash?

**Comment by rogerwang** [comment]
I can't reproduce this with 0.19 and a simple case.

**Comment by ghostoy** [comment]
@rogerwang Same result. It appears to be fixed in nw19.

@proclaim Could you test and confirm whether it's fixed in your case?

**Comment by proclaim** [comment]
@ghostoy yes, it's been resolved, thank you. (however, I still left the e.preventDefault() in there. Now I'm seeing something new in v19, I tried it to on 18.8 and didn't see any problem. here's the error output from my console

```
[4026:775:1205/102739:ERROR:gles2_cmd_decoder.cc(2379)] [.Offscreen-For-WebGL-0x7f9217142800]GL ERROR :GL_INVALID_VALUE : GLES2DecoderImpl::DoBindTexImage2DCHROMIUM: <- error from previous GL command
[4026:775:1205/102739:ERROR:gles2_cmd_decoder.cc(2379)] [.Offscreen-For-WebGL-0x7f9217142800]GL ERROR :GL_INVALID_VALUE : ScopedTextureBinder::ctor: <- error from previous GL command
[4026:775:1205/102739:ERROR:gles2_cmd_decoder.cc(2379)] [.Offscreen-For-WebGL-0x7f9217142800]GL ERROR :GL_INVALID_VALUE : ScopedTextureBinder::ctor: <- error from previous GL command
[4026:775:1205/102739:ERROR:gles2_cmd_decoder.cc(2379)] [.Offscreen-For-WebGL-0x7f9217142800]GL ERROR :GL_INVALID_VALUE : ScopedTextureBinder::ctor: <- error from previous GL command
[4026:775:1205/102739:ERROR:gles2_cmd_decoder.cc(2379)] [.Offscreen-For-WebGL-0x7f9217142800]GL ERROR :GL_INVALID_VALUE : GLES2DecoderImpl::DoBindTexImage2DCHROMIUM: <- error from previous GL command
[4026:775:1205/102739:ERROR:gles2_cmd_decoder.cc(2379)] [.Offscreen-For-WebGL-0x7f9217142800]GL ERROR :GL_INVALID_VALUE : ScopedTextureBinder::ctor: <- error from previous GL command
[4026:775:1205/102751:ERROR:gles2_cmd_decoder.cc(2379)] [.Offscreen-For-WebGL-0x7f9217142800]GL ERROR :GL_INVALID_VALUE : GLES2DecoderImpl::DoReleaseTexImage2DCHROMIUM: <- error from previous GL command
```

I'm suspecting this is a chronium bug instead of nwjs? please go ahead and close this ticket and open a new one if this error is valid

**Comment by ghostoy** [comment]
Thanks @proclaim 😄

**Comment by dalidong2009** [comment]
I'm using the nw 0.19.0 ia32 sdk version on Windows 2012 server 
as i quickly press twice F12 to open delTools, the app crash
I has used this version a few day, but this case only occur one time. and i use another version of nw, never occur this case.
here is my crash dump:

[crashdump.txt](https://github.com/nwjs/nw.js/files/641464/crashdump.txt)

...........................
Operating system: Windows NT
                  6.2.9200 16384
CPU: x86
     GenuineIntel family 6 model 42 stepping 1
     8 CPUs

Crash reason:  EXCEPTION_BREAKPOINT
Crash address: 0x702acec3

Thread 0 (crashed)
 0  nw.dll!`anonymous namespace'::InvalidParameter [process_startup_helper.cc : 21 + 0x0]
    eip = 0x702acec3   esp = 0x0023f338   ebp = 0x0023f338   ebx = 0x0023f568
    esi = 0x702acec0   edi = 0x72d9f4e0   eax = 0x702acec0   ecx = 0x702acec0
    edx = 0x72fab49c   efl = 0x00000206
    Found by: given as instruction pointer in context
 1  nw.dll!_purecall [purevirt.cpp : 23 + 0x2]
    eip = 0x71c692d0   esp = 0x0023f340   ebp = 0x0023f34c
    Found by: call frame info
 2  nw.dll!base::internal::Invoker<base::internal::BindState<void (*)(base::WaitableEvent *),base::internal::OwnedWrapper<base::WaitableEvent> >,void >::Run(base::internal::BindStateBase *) [bind_internal.h : 339 + 0x9]
    eip = 0x700c34c4   esp = 0x0023f348   ebp = 0x0023f34c
    Found by: call frame info
 3  nw.dll!base::debug::TaskAnnotator::RunTask(char const *,base::PendingTask const &) [task_annotator.cc : 54 + 0x18]
    eip = 0x702b9040   esp = 0x0023f354   ebp = 0x0023f3a8
    Found by: call frame info
 4  nw.dll!base::MessageLoop::RunTask(base::PendingTask const &) [message_loop.cc : 408 + 0x11]
    eip = 0x7026f21d   esp = 0x0023f3b0   ebp = 0x0023f4f0
    Found by: call frame info
 5  nw.dll!base::MessageLoop::DoWork() [message_loop.cc : 516 + 0xb]
    eip = 0x7026eb2e   esp = 0x0023f4f8   ebp = 0x0023f5b0
    Found by: call frame info
 6  nw.dll!base::MessagePumpForUI::DoRunLoop() [message_pump_win.cc : 263 + 0x9]
    eip = 0x702b9cea   esp = 0x0023f5b8   ebp = 0x0023f5e0
    Found by: call frame info
 7  nw.dll!base::MessagePumpWin::Run(base::MessagePump::Delegate *) [message_pump_win.cc : 141 + 0xa]
    eip = 0x702ba70a   esp = 0x0023f5e8   ebp = 0x0023f608
    Found by: call frame info
 8  nw.dll!base::MessageLoop::RunHandler() [message_loop.cc : 373 + 0x9]
    eip = 0x7026ef3b   esp = 0x0023f610   ebp = 0x0023f634
    Found by: call frame info
 9  nw.dll!base::RunLoop::Run() [run_loop.cc : 35 + 0x7]
    eip = 0x7029bc15   esp = 0x0023f618   ebp = 0x0023f634
    Found by: call frame info
10  nw.dll!ChromeBrowserMainParts::MainMessageLoopRun(int *) [chrome_browser_main.cc : 2133 + 0x8]
    eip = 0x7011ee3f   esp = 0x0023f63c   ebp = 0x0023f668
    Found by: call frame info
11  nw.dll!content::BrowserMainLoop::RunMainMessageLoopParts() [browser_main_loop.cc : 981 + 0x9]
    eip = 0x6fc3b795   esp = 0x0023f670   ebp = 0x0023f68c
    Found by: call frame info
12  nw.dll!content::BrowserMainRunnerImpl::Run() [browser_main_runner.cc : 155 + 0x8]
    eip = 0x6fc3c77f   esp = 0x0023f684   ebp = 0x0023f68c
    Found by: call frame info
13  nw.dll!content::BrowserMain(content::MainFunctionParams const &) [browser_main.cc : 46 + 0x7]
    eip = 0x6fc3888e   esp = 0x0023f694   ebp = 0x0023f6a4
    Found by: call frame info
14  nw.dll!content::RunNamedProcessTypeMain(std::basic_string<char,std::char_traits<char>,std::allocator<char> > const &,content::MainFunctionParams const &,content::ContentMainDelegate *) [content_main_runner.cc : 411 + 0xa]
    eip = 0x70061c33   esp = 0x0023f6ac   ebp = 0x0023f6b8
    Found by: call frame info
15  nw.dll!content::ContentMainRunnerImpl::Run() [content_main_runner.cc : 782 + 0x6]
    eip = 0x70061b83   esp = 0x0023f6c0   ebp = 0x0023f720
    Found by: call frame info
16  nw.dll!content::ContentMain(content::ContentMainParams const &) [content_main.cc : 22 + 0x7]
    eip = 0x7006122b   esp = 0x0023f728   ebp = 0x0023f730
    Found by: call frame info
17  nw.dll!ChromeMain [chrome_main.cc : 97 + 0x9]
    eip = 0x6f3b36c2   esp = 0x0023f738   ebp = 0x0023f82c
    Found by: call frame info
18  nw.exe!MainDllLoader::Launch(HINSTANCE__ *,base::TimeTicks) [main_dll_loader_win.cc : 185 + 0xf]
    eip = 0x00ea44a9   esp = 0x0023f834   ebp = 0x0023f8d0
    Found by: call frame info
19  nw.exe!wWinMain [chrome_exe_main_win.cc : 255 + 0x10]
    eip = 0x00ea27ca   esp = 0x0023f8d8   ebp = 0x0023fa08
    Found by: call frame info
20  nw.exe!__scrt_common_main_seh [exe_common.inl : 255 + 0x1a]
    eip = 0x00ee34f4   esp = 0x0023fa10   ebp = 0x0023fa54
    Found by: call frame info
21  KERNEL32.DLL + 0x28543
    eip = 0x761f8543   esp = 0x0023fa5c   ebp = 0x0023fa60
    Found by: call frame info
22  ntdll.dll + 0x5ac69
    eip = 0x77bfac69   esp = 0x0023fa68   ebp = 0x0023faa4
    Found by: previous frame's frame pointer
23  ntdll.dll + 0x5ac3c
    eip = 0x77bfac3c   esp = 0x0023faac   ebp = 0x0023fabc
    Found by: previous frame's frame pointer
......................................

**Comment by ghostoy** [comment]
@dalidong2009 I cannot reproduce this issue. And your crash report isn't the same as this issue. Please create another issue and we will look into it.

**Comment by dalidong2009** [comment]
OK , in #5536
